api/src/main/java/org/neo4j/ogm/config/ConfigurationUtils.java

```diff
@@ -71,6 +90,11 @@ private static URL getClasspathResourceUrl(String path) throws FileNotFoundExcep
         return url;
     }
 
-    private ResourceUtils() {
+    private static String createRelativePathIfNecessary(String path) {
+
+        return path.startsWith("/") ? path.substring(1) : path;
```

Comment:
Maybe refactor this to something more obvious to express the removal of the `/`.

Reply:
Nope.